tests/test_model.py

```diff
@@ -438,36 +437,6 @@ def test_loss_fn():
                                     atol=1e-4), f'differed at step {i}'
 
 
-@pytest.mark.parametrize('prefixlm', [False, True])
-def test_opt_wrapping(prefixlm: bool):
```

Comment:
I don't know that we want to axe this test, unless we're giving up on OPT support entirely. I'm pretty sure this test actually is more to do with FSDP wrapping being correct, so we'd just want to slice away the Prefix LM part of it.

Reply:
you are correct. fixed.